tests/Integration/Behaviour/Features/Scenario/Cart/Adding/Product/add_standard_product.feature

```diff
@@ -26,7 +26,7 @@ Feature: Add product in cart
     And  I create an empty cart "dummy_cart" for customer "testCustomer"
     And there is a product in the catalog named "product1" with a price of 19.812 and 1000 items in stock
     And the product "product1" minimal quantity is 10
-    When I add 5 products "product1" to the cart "dummy_cart"
+    When I update quantity of product "product1" in the cart "dummy_cart" to 5
```

Comment:
Why "update quantity"? It makes no sense in natural language. It would if the products were already in the cart.

Reply:
Yes. Wrong replacement. There was many cases where it was an update and `add` was used. But not this one ;)